spk/ffmpeg/src/installer.sh

```diff
@@ -2,10 +2,11 @@
 
 # Package
 PACKAGE="ffmpeg"
+DNAME="ffmpeg"
 
-FFMPEG_TARGET="/usr/bin/${PACKAGE}"
-FFPROBE_TARGET="/usr/bin/ffprobe"
-FFSERVER_TARGET="/usr/bin/ffserver"
+FFMPEG_TARGET="/usr/local/bin/${PACKAGE}"
```

Comment:
Isn't it `${INSTALL_DIR}/bin/ffmpeg`, etc, if anything? Besides, imo we don't want to add ffmpeg to PATH, to prevent issues with other ffmpeg binaries floating around on the system.